packages/ckeditor5-ckbox/src/ckboxcommand.ts

```diff
@@ -424,12 +428,16 @@ export function prepareImageAssetAttributes( asset: CKBoxRawAssetDefinition ): C
 /**
  * Parses the assets attributes into the internal data format.
  *
- * @param origin The base URL for assets inserted into the editor.
+ * @param config The CKBox download asset configuration.
+ * @param asset The asset to prepare the attributes for.
  */
-function prepareLinkAssetAttributes( asset: CKBoxRawAssetDefinition ): CKBoxAssetLinkAttributesDefinition {
+function prepareLinkAssetAttributes(
+	config: CKBoxConfig[ 'downloadableFiles' ],
+	asset: CKBoxRawAssetDefinition
+): CKBoxAssetLinkAttributesDefinition {
```

Comment:
Although it doesn't change the logic, I prefer to put the most important parameters first and the optional ones last to make the function clearer and easier to use. `config` is here optional and `asset` seems to be more important. Therefore I'd change the order of parameters to `( asset, config )`.